stmt/stmts/update.go

```diff
@@ -279,7 +296,7 @@ func (s *UpdateStmt) Exec(ctx context.Context) (_ rset.Recordset, err error) {
 			end := start + len(tbl.Cols())
 			data := rowData[start:end]
 			start = end
-			tcols, err2 := getUpdateColumns(tbl, s.List, isMultipleTable)
+			tcols, err2 := getUpdateColumns(tbl, s.List, isMultipleTable, tblAliasMap)
```

Comment:
why not use `GetFieldIndex` or others in fields package.

I thin these can do the same thing, @qiuyesuifeng, how do you think?